test/handler/FileHandlerTest.cpp

```diff
@@ -51,37 +52,39 @@ TEST_CASE_METHOD(FileHandlerTestUtils, "FileHandler Testing", "[FileHandler]") {
 
     SECTION("Check file saving") {
         auto *const tableWidget = new QTableWidget(2, 6);
+
         auto *const checkBoxFighter = new QCheckBox;
         checkBoxFighter->setChecked(false);
         auto *const checkBoxBoss = new QCheckBox;
         checkBoxBoss->setChecked(true);
 
-        auto *const widgetFighter = new QWidget;
-        // Centering fighter checkbox
-        auto *layoutFighter = new QHBoxLayout(widgetFighter);
-        layoutFighter->addWidget(checkBoxFighter);
-        widgetFighter->setLayout(layoutFighter);
+        auto *const additionalInfoWidgetFighter = new AdditionalInfoWidget;
+        additionalInfoWidgetFighter->setMainInfoText("Haste");
+        auto *const additionalInfoWidgetBoss = new AdditionalInfoWidget;
+
+        const auto createCellWidget = [&] (QWidget* widget) {
```

Comment:
Better set a pointer so it is more easily understandable that a QWidget is returned